products-built-on-oxen/lokinet/guides/lokinet-on-a-soho-router.md

```diff
@@ -0,0 +1,325 @@
+
+`# apt update`
+
+now install the lokinet package this way:
+
+`# apt install lokinet --no-install-recommends`
+
+#### set up lokinet configs
+
+the easiest way to configure lokinet is to use config override files and not touch `lokineti.ini` at all.
+that directory we try to load from if it exists is located at `/var/lib/lokinet/conf.d`. by default as of `0.9.11` tag it does not get autocreated.
+
+so first, we create the config override dir:
+
+`# mkdir -p /var/lib/lokinet/conf.d`
+
+the first config override we want contains the network settings for lokinet for things like dns, ip range and interface name. this example will make lokinet only resolve `.loki` and `.snode`
+
+put in config overrides into `/var/lib/lokinet/conf.d/wifi.ini`
+
+file contents:
+
+```ini
+[network]
+ifname=lokinet0
+ifaddr=10.100.0.1/16
+
+[dns]
+bind=127.0.42.1:5533
+# when this option is emtpy we only resolve .loki and .snode and will SERVFAIL any other queries.
+upstream=
+
+```
+
+if you want to use an exit node make a new file at `/var/lib/lokinet/conf.d/exit-node.ini`
+
+file contents:
+
+```ini
+[network]
+exit-node=exit.loki
+exit-node=exit2.loki
+```
+
+you can swap out exit.loki and exit2.loki with any other exit you know of, these are provided as functionging placeholder/default values for toy usage, the OPTF runs both of these exits at the moment.
+
+you can add as many entries as you want and it will stripe traffic accross them per remote ip address.
+
+then edit the systemd unit to add a post up command we'll need for the routing table:
+
+`# systemctl edit lokinet`
+
+this will enter an editor. add the following snippet affer the the comment block where it tells you in the editor:
+
+```ini
+[Service]
+PostExecStart=+/sbin/iptables -A POSTROUTING -j MASQUERADE -o lokinet0
+```
+
+then save and quit that editor (in nano it's control + o then enter then control + x).
+
+### hostapd
+
+this daemon manages the wireless access point.
+
+#### configuration
+
+in this example we set the ssid of the wifi network to be `lokinet` with the very strong password of `lokipass` (for real networks use a better password).
+you MUST also set your country code, for the usa it's `US` for australia it is `AU` and for germany it is `DE`.
+
+see the debian wiki page on [hostapd](https://wiki.debian.org/hostap) for more help on configuring.
+
+config file at `/etc/hostapd/hostapd.conf`:
+
+```ini
+interface=wifi0
+hw_mode=g
+channel=0
+ieee80211d=1
+ieee80211n=1
+ieee80211ac=1         
+wmm_enabled=1
+auth_algs=1
+wpa=2
+wpa_key_mgmt=WPA-PSK
+rsn_pairwise=CCMP
+
+ssid=lokinet
+wpa_passphrase=lokipass
+
+;; country_code=[insert your country code here]
+
+```
+
+#### caveats
+
+many settings of hostapd may need to change depending on your hardware and region.
+you will likely need to consult the main documentation for hostapd if you have more troubles.
+
+### unbound
```

Comment:
I think this belongs as an "extra", i.e. "here's how you can set up DNS on the network to go through adguard's DNS over TLS", because if you leave it out you can just point dnsmasq at your existing router which significantly simplifies the configuration (and then set up adguard DNS or whatever on *that* router).

Reply:
i.e. document people how to do the *simple* setup first, before making it more complicated.